include/wx/xrc/xh_styledtextctrl.h

```diff
@@ -0,0 +1,25 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        wx/xrc/xh_styledtextctrl.h
+// Purpose:     XML resource handler for wxStyledTextCtrl
+/////////////////////////////////////////////////////////////////////////////
```

Comment:
```suggestion
// Copyright:   (c) 2021 TT-Solutions SARL
// Licence:     wxWindows licence
/////////////////////////////////////////////////////////////////////////////
```

Reply:
Header fields _Author_ and _Created_ are not used anymore (relying on GIT history instead)? If so, perhaps the coding guidelines should be updated, see http://wxwidgets.org/develop/coding-guidelines/#wxwin_header.